engine/crates/engine/registry-v2/src/resolvers/postgres.rs

```diff
@@ -0,0 +1,43 @@
+// #[serde_with::minify_field_names(serialize = "minified", deserialize = "minified")]
```

Comment:
I see a bunch of these annotations. Was the idea abandoned? I don't know what it does but the generated code with numbers for field names looks pretty compact already. In JSON I think it could only get more compact by serializing to an array instead of objects, but it would probably be more fragile.

Reply:
Oh, this being commented out is an oversight.  I copied & pasted all the resolvers over from engine, but then commented out the serde_with stuff initially so I could get it working.

I've ended up adding serde_with back in for the types that aren't yet otherwise compact, but I obviously missed this one.

I _think_ I want to eventually move the resolver heirarchy into the generated code, which would remove the need for these annotations.  But not planning on doing that immediately, it's a medium sized job.